pkg/httpclient/client.go

```diff
@@ -176,6 +176,16 @@ func (c *Client) Post(path string, contentType string, body io.Reader, opts ...O
 	return c.Do(req)
 }
 
+// Put issues a PUT to the specified DC/OS cluster path.
+func (c *Client) Put(path string, contentType string, body io.Reader, opts ...Option) (*http.Response, error) {
+	req, err := c.NewRequest(http.MethodPut, path, body, opts...)
```

Comment:
nit: can you use `PUT`, or update other methods to use the constant? I'm fine with either, as long as it's consistent.